src/idl_gen_grpc.cpp

```diff
@@ -114,7 +114,8 @@ class FlatBufPrinter : public grpc_cpp_generator::Printer {
       // See if this contains more than one line.
       auto lf = strchr(s, '\n');
       if (lf) {
-        (*str_) += std::string(s, lf + 1);
+        auto index = (int)(lf-s)+1;
```

Comment:
Thanks for the quick fix! This has a few very minor issues though:
- `index` is a bit generic, `line_length` maybe?
- use C++ casts like `static_cast`
- type needs to be `size_t`, not `int`
- spacing around `+` and `-`